main.py

```diff
@@ -0,0 +1,58 @@
+from dotenv import load_dotenv
```

Comment:
### About ETL Script

This script is fixed to one task.  It will require some work to reuse this for other task.